Framework/Algorithms/src/PolarizationEfficiencyCor.cpp

```diff
@@ -26,13 +27,16 @@ namespace {
 /// Property names.
 namespace Prop {
 static const std::string FLIPPERS{"Flippers"};
-static const std::string SPIN_STATES{"SpinStates"};
+static const std::string SPIN_STATES{"SpinStatesOutWildes"};
```

Comment:
We should rename this variable to clearly differenciate it from the Fredrikze ones.
```suggestion
static const std::string OUTPUT_WILDES_SPIN_STATES{"SpinStatesOutWildes"};
```